medusa/providers/torrent/html/hdspace.py

```diff
@@ -25,17 +25,18 @@
 
 from requests.compat import urljoin
 from requests.utils import dict_from_cookiejar
+
 from ..TorrentProvider import TorrentProvider
 from .... import logger, tvcache
 from ....bs4_parser import BS4Parser
 from ....helper.common import convert_size, try_int
 
 
-class HDSpaceProvider(TorrentProvider):  # pylint: disable=too-many-instance-attributes
```

Comment:
@medariox need to update pytest.ini right?

@ratoaq2 can we run your script to update it after so many codacy fixes ?